src/pages/userinfo/edit/index.tsx

```diff
@@ -1,19 +1,89 @@
-import type { NextPage } from 'next';
-import Head from 'next/head';
-import React from 'react';
-
-const UserinfoEdit: NextPage = () => {
-  return (
-    <React.Fragment>
-      <Head>
-        <title>우리의 여행코스 | 이곳저곳</title>
-        <meta name="description" content="our travel course" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
-
-      <main>UserinfoEdit</main>
-    </React.Fragment>
-  );
-};
-
-export default UserinfoEdit;
+import styled from '@emotion/styled';
+import type { NextPage } from 'next';
+import Head from 'next/head';
+import React from 'react';
+import { Button, PageContainer, Text, Title } from '~/components/atom';
+import theme from '~/styles/theme';
+
+const UserinfoEdit: NextPage = () => {
+  return (
+    <React.Fragment>
+      <Head>
+        <title>우리의 여행코스 | 이곳저곳</title>
+        <meta name="description" content="our travel course" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <main>
+        <PageContainer>
+          <Container>
+            <Title>내 정보 수정</Title>
+            <Form>
+              <FormItem>
+                <FormTitle>
+                  <Text>이메일</Text>
+                </FormTitle>
+                <FormGroup>
+                  <input type="email" name="email" />
+                </FormGroup>
+              </FormItem>
+              <FormItem>
+                <FormTitle>
+                  <Text>닉네임</Text>
+                </FormTitle>
+                <FormGroup>
+                  <input type="text" name="nickname" />
+                </FormGroup>
+              </FormItem>
+              <FormItem>
+                <FormTitle>
+                  <Text>성별</Text>
+                </FormTitle>
+                <FormGroup>
+                  <input type="text" name="sex" />
+                </FormGroup>
+              </FormItem>
+              <FormItem>
+                <FormTitle>
+                  <Text>생년월일</Text>
+                </FormTitle>
+                <FormGroup>
+                  <input type="email" name="birth" />

```

Comment:
```suggestion
                  <input type="date" name="birth" />
```
요기 type `date`로 바꿔주셔야 될거같아요!

Reply:
아 그러네요! 😀 수정하겠습니다~!